lxd/migrate_storage_volumes.go

```diff
@@ -50,7 +50,7 @@ func (s *migrationSourceWs) DoStorage(state *state.State, poolName string, volNa
 			return err
 		}
 
-		poolMigrationTypes = pool.MigrationTypes(storageDrivers.ContentTypeFS)
+		poolMigrationTypes = pool.MigrationTypes(storageDrivers.ContentTypeFS, false)
```

Comment:
@monstermunchkin same point as above about this being dynamic based on whether the migration is a refresh or not.

Reply:
@tomponline Storage volume migration doesn't support refresh.

Reply:
@monstermunchkin ah fair enough then :)